### PR TITLE
Feature/ Add contributor names to readme for cohort. Final edits.

### PR DESCRIPTION
**Pull request by rhiannonstanford**
Pair Programmed with Julian Isrel

### CODE REVIEW

**Review by rhiannonstanford — COMMENTED** [review]
Self review.

**Review by artofmayhem — APPROVED** [review]
Sharp looking addition!

File: README.md [hunk 1]
```diff
@@ -75,12 +75,28 @@ Scribble Stadium is a paid service; parents are required to create the account f
...
+|                                                                     Technical Lead Engineer                                                                     |                                                                  Team Project Lead                                                                   |                                                                  Lead Backend Engineer                                                                  |                                                                     Frontend Engineer                                                                     |                                                                  UX Engineer                                                                   |                                                           Lead UX Engineer                                                           |
```
Comment (Julianisrel) [inline README.md:78]: Looks nice and clean!

**Review by Julianisrel — APPROVED** [review]

**Review by Julianisrel — APPROVED** [review]
Self Review.

**Review by artofmayhem — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:78]: Thanks for catching this. I know I changed this myself; I even sent you a snapshot, but with all the copying and pasting I was doing, I must have gotten mixed up somewhere.

**Review by ghost — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rhiannonstanford) [reply README.md:78]: You did great.  I am Always forgetting to save the file.  I do this all the time.

**Pull request merged**